src/commands/__snapshots__/help.test.ts.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
What's this extra file all about? And the weird `__snapshots__` subdirectory? Clutter?

Reply:
That's a smapshot file, generated by Jest the first time a snapshot is taken. Future tests will compare outputs against this file, and yell if things change unexpectedly, indicating a failed test.